_posts/2018-08-10-links-to-ddd-perth.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Links to DDD Perth talks"
+date:   2018-08-10
+tags: serverless cloud native pwa webapps
+author: William Sia
+image: img/native-mobile.png
+---
+
+On the 4th of August I had a great privilege to do a talk on [DDD Perth](https://dddperth.com/), however I foolishly submitted the talk for the 20 minutes, rather than the 45 minutes slot. As a result, we don't have a lot of time to go through the content in more details. This is my attempt to provide more information related to the talk. Come to our office <img src="https://static.thenounproject.com/png/894284-200.png" width="24"> **Mech Rock HQ Suite 2, 51 King Street, Perth WA 6000** for a quick chat.
```

Comment:
On the 4th of August I had the great privilege to do a talk _at_ [DDD Perth](https://dddperth.com/), however I foolishly submitted the talk for the 20 minute track, rather than a 45 minute slot. As a result, we didn't have a lot of time to go through the content in more detail. This is my attempt to provide more information related to the talk. Come to our office <img src="https://static.thenounproject.com/png/894284-200.png" width="24"> **Mech Rock HQ Suite 2, 51 King Street, Perth WA 6000** for a quick chat and a game of table tennis.